pxr/usd/usdGeom/schema.usda

```diff
@@ -2353,16 +2353,38 @@ class Camera "Camera" (
         doc = """Frame relative shutter close time, analogous comments from
                  shutter:open apply. A value greater or equal to shutter:open
                  should be authored, otherwise there is no exposure and a
-                 renderer should produce a black image."""
+                 renderer should produce a black image. Used for motion blur."""
     )
     
```

Comment:
In the larger physical camera proposal, we've got this chunk of verbiage ~

```
Describes the exposure settings common to still and cinema cameras.

    These settings are used to scale the brightness of the illumination incident
    on the camera system. The scaling factor, S, is computed as:
       S = exposure_time * iso * 2^exposure_compensation * responsivity_compensation / (100 * fnumber^2)
```

Can we work that in somewhere?

Reply:
Can't the existing exposure be used as exposure_compensation? If all the others have appropriate fallback values then the result will be the exact same scaling factor without any need to switch between different math expressions.

Reply:
Nevermind, I see that was in the original proposal. The default (sorry, the "fallback") values for all the attributes are apparently set so that the math is 2^exposure.

I do recommend not using the term "exposure_compensation" anywhere, use "exposure" instead, so nobody is confused into thinking there are more parameters than there really are.

Reply:
Yeah can rename it here. 

I do use exposureCompensation as a token in HdCamera in order to be able to read the `exposure` attribute as opposed to getting the computed exposure value.

Reply:
@meshula I've added a bit of an explanation to the UsdGeomCamera description which renders like the attached
![Screenshot_20240516_120837](https://github.com/PixarAnimationStudios/OpenUSD/assets/1935453/3ef76cf9-ad91-4d83-b21f-4d7dc42b1443)

If you think that fits the bill I'll push it

Reply:
Yes, please :)

Reply:
That's awesome, thank you!!